brambling/templates/brambling/event/order/_attendee_summary.html

```diff
@@ -25,10 +25,13 @@
 							({{ item.item_name }})
 							{% if event.transfers_allowed %}
 								{% url "brambling_event_order_transfer" event_slug=event.slug organization_slug=event.organization.slug as transfer_url %}
-								{% if item.status == item.BOUGHT %}
+								{% if item.can_transfer %}
```

Comment:
I would like to register a concern that `item.can_transfer` always triggers a query. For now this shouldn't be an issue because this is only being used once on this page, but if you _wanted_ to cache the property I wouldn't object ;-)